app/views/components/input_radio.scala.html

```diff
@@ -28,7 +28,7 @@
 )(implicit messages: Messages)
 
 <div class="form-group @if(field.hasErrors){form-group-error}">
-    <fieldset @if(fieldSetClass.nonEmpty){class="@fieldSetClass.get"} id="@{field.id}" @if(field.hasErrors || hint.nonEmpty){aria-describedby="@if(field.hasErrors){error-message-@{field.id}-input} @if(hint.nonEmpty){hint-@{field.id}}"}>
+    <fieldset @if(fieldSetClass.nonEmpty){class="@fieldSetClass.get"} id="@{field.id}" @if(field.hasErrors || hint.nonEmpty){aria-describedby="@if(field.hasErrors){error-message-@{field.id}-input}@if(field.hasErrors && hint.nonEmpty){ }@if(hint.nonEmpty){hint-@{field.id}}"}>
         @if(headingIsLegend){
```

Comment:
This new condition does nothing ?

Reply:
It does, it only puts a space in if both exist, that fixes he aria-describedby issue. It's what we've done in all the trusts services.

Reply:
ok cool